recipe/meta.yaml

```
@@ -17,8 +17,9 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
-    - setuptools_scm
+# Lower bound for setuptools not required but recommended by upstream
+    - setuptools >=61.0.0
+    - setuptools_scm >=6.2
```

https://github.com/penguinolog/sqlalchemy_jsonfield/blob/1.0.2/pyproject.toml#L6
setuptools_scm[toml]>=6.2
```suggestion
    - setuptools_scm >=6.2
    - toml
```